web/templates/popovers/left_sidebar/left_sidebar_topic_actions_popover.hbs

```diff
@@ -107,6 +107,14 @@
                 </a>
             </li>
             {{/if}}
+            {{#if is_realm_admin}}
```

Comment:
I think ideally we want to offer this to moderators too.

Reply:
Done.